magento/pwa-studio
# [Docs] Internationalization docs

## Description

Creates a new Internationalization topic that provides an overview of the I18n framework in PWA Studio.

## Related Issue

Closes PWA-930

Related to work done in #2696 

## Acceptance 

@davemacaulay or anyone in the owls team
Any other untitled geese developer

### Verification Stakeholders

@davemacaulay or anyone in the owls team
Any other untitled geese developer

### Specification



### Verification Steps

1. Navigate to the pwa-devdocs directory: `cd pwa-devdocs`
1. Start the html preview server: `yarn develop`
1. Navigate to: `/technologies/basic-concepts/internationalization/`
1. Verify page exists
1. Verify navigation entry is highlighted on the left navigation

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
